custom.js

```diff
@@ -304,16 +304,20 @@
 		getData( api.url, processData, api.err);
 
 		// Toggle between calendar and list views
-		$( "#weekbutton" ).on('click', function() {     
-			$('.listview').hide();
-			$('.weekview').show();
-			$('.dots').show();
-		});
-
-		$( "#listbutton" ).on('click', function() {     
-			$('.weekview').hide();
-			$('.listview').show();
-			$('.dots').hide();
+		
+		$('.views').on('click', '.view', function() {
+			$(this).addClass('showing').siblings().removeClass('showing');
+
+			if (this.id === 'weekbutton') {
```

Comment:
I think you can use a similar approach as you did on line 309 to determine what gets shown.

Reply:
By this I mean, you can simply add a class to the active or shown view. Maybe wrap both views in their own containers which would house the view.
```
div.viewlinks
  a.viewlink
  a.viewlink

div.view.listview.active
  .list
div.view.weekview
  .week
  .dots
```

You can then simply change the class for the active view and everything inside it will show.